source/playgrounds.txt

```diff
@@ -150,19 +150,41 @@ If you have a playground file open in |vscode-short| and do not have
 an active connection, |vsce| displays
 :guilabel:`Click here to add connection` at the top of your playground.
 
-.. figure:: /images/vsce-playground-add-connection-btn.png
-   :figwidth: 700px
-   :alt: Link to add connection from playground
-
-1. Click this link to open the connection string dialog.
-
-#. Enter the
-   :manual:`connection string </reference/connection-string/>` for the
-   deployment you want to run this playground against.
+.. procedure::
+   :style: normal
    
-#. Press :guilabel:`Enter`.
-
-#. Run your playground.
+   .. step:: Select :guilabel:`Click here to add connection` 
+
+      After you click :guilabel:`Click here to add connection`, |vsce| opens the 
+      connection string dialog.
+
+      .. figure:: /images/vsce-playground-add-connection-btn.png
+         :figwidth: 700px
+         :alt: Link to add connection from playground
+         
+   .. step:: Enter your connection string
+      
+      Enter the :manual:`connection string </reference/connection-string/>` for 
```

Comment:
**Suggestion**

It may be valuable to enter a small sample connection string here to avoid the users needing to go to the additional page WDYT?
Something like:

```
mongodb://localhost:27017/mydatabase
```

Reply:
I'm leaning towards keeping this as is because I'm expecting that a user coming to use the VSCode extension playground would be familiar with MDB basics such as using a connection string, given they must already have a deployment set up.